lib/kube/proxy/resource_rbac_test.go

```diff
@@ -36,12 +36,16 @@ import (
 	authv1 "k8s.io/api/rbac/v1"
 	kubeerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer/streaming"
 	kubetypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/watch"
 	restclientwatch "k8s.io/client-go/rest/watch"
 
+	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Move to the block above and sort imports?

```suggestion
	controllerclient "sigs.k8s.io/controller-runtime/pkg/client"
```